src/websocket.c

```diff
@@ -383,15 +383,35 @@ void ws_read_cb(uv_link_t *l, ssize_t nread, const uv_buf_t *buf) {
     free(buf->base);
 }
 
-static void send_pong(um_websocket_t *ws, const char* ping_data, int len) {
+static void send_pong(um_websocket_t *ws, const char* ping_data, int len, bool masked, uint8_t mask[4]) {
     uv_buf_t buf;
     buf.len = 2 + len;
+    if (masked) {
+        buf.len += sizeof(mask);
+    }
     buf.base = malloc(buf.len);
 
     buf.base[0] = WS_FIN | OpCode_Pong;
     buf.base[1] = (char)(WS_MASK | (0x7f & len));
```

Comment:
`WS_MASK` flag is always set so `masked` parameter should also be always be true, right?